web.test.tests/web.test.tests/Tests/SettingsPageTests.cs

```diff
@@ -22,12 +23,11 @@ public void Setup()
 
             driver.Url = "http://localhost:64177/Login";
 
-            LoginPage loginPage = new LoginPage(driver);
-            loginPage.Login("test", "newyork1");
+            new LoginPage(driver).Login("test", "newyork1");
 
             new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("amount")));
 
-            driver.FindElement(By.XPath("//div[text()='Settings']")).Click();
+            new DepositPage(driver).Settings.Click();
             new WebDriverWait(driver, TimeSpan.FromMilliseconds(10000)).Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.XPath("//div[text()='Logout']")));
```

Comment:
This can be simplified by re-using methods:

new LoginPage(driver).Login().OpenSettingsPage();

Reply:
Done